src/pages/Success/Success.js

```diff
@@ -48,9 +47,12 @@ class Success extends Component {
               Comic Relief currently supports more than 2,000 projects in the UK and around
               the world, tackling issues like gender injustice and mental illness.
             </p>
-            <p>These are some of the people we helped last year:</p>
           </div>
-          <Cards />
+          <div>
+            <a href="https://www.comicrelief.com/your-impact" className="btn btn--red">
```

Comment:
could you place this div with the button inside the div with the class `success-wrapper--inner` (otherwise the button will full width)